promoteDevToStaging.sh

```diff
@@ -1,3 +1,8 @@
 #!/bin/bash
 
+# WARNING: This will apply CURRENT LOCAL migrations to the staging server,
+# which may not necessarily be in sync with the dev server.
+cd hasura/user-prefs
+hasura migrate apply --endpoint https://user-prefs-staging.herokuapp.com 
```

Comment:
Do we also need to run `hasura metadata apply ...` and maybe `hasura metadata reload`?